petab/observables.py

```diff
@@ -0,0 +1,40 @@
+"""Functions for working with the PEtab observables table"""
+
+import pandas as pd
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pandas'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4885513)